pkg/openstack/loadbalancer.go

```diff
@@ -687,13 +688,14 @@ func getBoolFromServiceAnnotation(service *corev1.Service, annotationKey string,
 		case "false":
 			returnValue = false
 		default:
+			klog.Error("Invalid Service Annotation: %v = %v (falling back to default setting: %v)", annotationKey, annotationValue, defaultSetting)
 			returnValue = defaultSetting
 		}
 
 		klog.V(4).Infof("Found a Service Annotation: %v = %v", annotationKey, returnValue)
 		return returnValue
 	}
-	klog.V(4).Infof("Could not find a Service Annotation; falling back to default setting: %v = %v", annotationKey, defaultSetting)
+	klog.V(4).Infof("Found a non-boolean Service Annotation; falling back to default setting: %v = %v", annotationKey, defaultSetting)
```

Comment:
I don't think it's the right place for non-boolean, if the func execution ever reaches here. It means that we couldn't find any service annotation for the following key. The log message currently indicates that if found an annotation when it didn't.